snafu/benchmarks/uperf/uperf.py

```diff
@@ -279,7 +279,7 @@ def get_results_from_stdout(stdout: UperfStdout) -> List[UperfStat]:
                 norm_ltcy = ((timestamp - prev_timestamp) / norm_ops) * 1000
 
             datapoint = UperfStat(
-                uperf_ts=datetime.datetime.fromtimestamp(int(timestamp) / 1000).isoformat(),
+                timestamp=datetime.datetime.now(),
```

Comment:
This is incorrect. Please review https://github.com/cloud-bulldozer/benchmark-wrapper/blob/79d54b37bfa8fd75283579eebedfba106b23f70c/snafu/benchmarks/uperf/uperf.py#L228-L240